### PR TITLE
Added agentsay and agentsvg to contrib

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @vpodzime can review this?

**Comment by olehermanse** [comment]
This pull request **introduces 2 alerts** when merging 69522a2d52c046f7fdb48bcfe154033984813a78 into 002d96f636f2db2b59713c90c2bde2792a11b2f5 - [view on LGTM.com](https://lgtm.com/projects/g/cfengine/core/rev/pr-923cdb322e973792e06637046a798d3af4d355a6)

**new alerts:**

* 2 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: contrib/agentsay/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+|             ●   ●             |
+|             ●   ●             |
+|                               |
+\-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-/
+```
+
+## Download
+```
+git clone https://github.com/olehermanse/agentsay.git
+cd agentsay
```
Comment (craigcomstock) [inline contrib/agentsay/README.md]: change Download section to reflect new location?

File: contrib/agentsay/agentsay.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""agentsay - print messages as an agent"""
+
+__copyright__ = "Northern.tech"
+__authors__ = ["Ole Herman Schumacher Elgesem"]
+
+# Install (as root):
+# curl -L https://raw.githubusercontent.com/olehermanse/agentsay/master/agentsay.py -o /usr/local/bin/agentsay && chmod ugo+x /usr/local/bin/agentsay && agentsay install success
+
```
Comment (craigcomstock) [inline contrib/agentsay/agentsay.py]: fixup this comment's URL as well?

File: contrib/agentsay/README.md (hunk 1 above)
Comment (olehermanse) [reply contrib/agentsay/README.md]: Can't test without merging this first ;) Thanks for pointing it out, I'll create a follow up.

**Pull request merged**